<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserDialog</class>
 <widget class="QDialog" name="UserDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Benutzer-Dialog</string>
  </property>
  <widget class="QTableView" name="userTable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>781</width>
     <height>311</height>
    </rect>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="userModifyButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Benutzer ändern</string>
   </property>
  </widget>
  <widget class="QPushButton" name="userDeleteButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>330</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Benutzer löschen</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
